compiler/luci/pass/include/luci/Pass/MakeBatchNormGammaPositivePass.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __LUCI_MAKE_BATCH_NORM_GAMMA_POSITIVE_PASS_H__
+#define __LUCI_MAKE_BATCH_NORM_GAMMA_POSITIVE_PASS_H__
+
+#include <logo/Pass.h>
+
+namespace luci
+{
+
+/**
+ * @brief  Class to make negative gamma of batchnorm to a small positive value (1e-10)
+ *         This pass can change the execution result of the model.
+ *         So, use it only when the impact is known to be acceptable.
+ */
```

Comment:
It doesn't have to be resolved in this PR, shouldn't this be provided as a help description of the options in `circle-optimizer` CLI?

Reply:
I agree. I'll add this description in the option of `circle-optimizer` as well.